neptune/new/exceptions.py

```diff
@@ -1131,3 +1131,26 @@ def __init__(self, missing_feature):
 """
         self.message = message.format(missing_feature=missing_feature, **STYLES)
         super().__init__(message)
+
+
+class NeptuneExperimentCreationConflict(NeptuneException):
```

Comment:
if uncaught, this exception can reach user -- so I'm not very enthusiastic about using the "Experiment" word

Reply:
Any suggestion?

Reply:
I think my preferred course of action would be to just:
1. delete `NeptuneExperimentCreationConflict` altogether, let `HTTPConflict` fly from `HostedNeptuneBackend`
2. make `NeptuneKeyAlreadyExistsError` (after rename, possibly - cf. other comment ;)) simply a subclass of `NeptuneException`

Reply:
> if uncaught, this exception can reach a user

Same for HTTPConflict :wink:. 

> let `HTTPConflict` fly from `HostedNeptuneBackend`

I'm not a big fan as the main idea was to overwrap the HttpConflict with something to hide the implementation details. HostedNeptuneBackend is only implementation and you would like to use part of that in the "upper layer".  I could imagine that for instance, OfflineNeptuneBackend could implement such logic and what it should raise? HttpConflict?

Reply:
> Same for HTTPConflict
yep, but that's a legit exception, not our weird invention ;)

as to rest -- weeeell, you do have a point
`NeptuneObjectCreationConflict`? :)